docs/faking-async-methods.md

```diff
@@ -27,3 +27,19 @@ Of course, you can still configure calls to `async` methods as you would normall
 ```csharp
 A.CallTo(() => foo.Bar()).Returns(Task.FromResult("bar"));
 ```
+
+There are also convenience overloads of `Returns` and `ReturnsLazily` that let you specify a value rather
+than a task, and configure the method to return a completed task whose result is the specified value:
+
+```csharp
+A.CallTo(() => foo.Bar()).Returns("bar");
+```
+
+These overloads of `Returns` and `ReturnsLazily` also exist for `ValueTask<T>`; however, unless you're
+using the .NET Standard 2.1 version of FakeItEasy, they're in a separate package:
+[`FakeItEasy.Extensions.ValueTask`](https://www.nuget.org/packages/FakeItEasy.Extensions.ValueTask).
+
+## Throwing exceptions
+
+To configure an async method to throw an exception, see
+[Throwing exceptions from an async method](throwing-exceptions.md#throwing-exceptions-from-an-async-method).

```

Comment:
Not sure if *Throwing exceptions from an async method* belongs on this page, or on "Throwing exceptions"... So I just added a link.